model/src/main/proto/topic.proto

```diff
@@ -257,6 +257,58 @@ message TopicProgress {
 
   // Map from story ID to StoryProgress.
   map<string, StoryProgress> story_progress = 2;
+
+  // Timestamp to record last time the exploration was played in ms.
+  int64 last_played_timestamp = 3;
```

Comment:
@veena14cs Any updates on this?

Reply:
It is difficult to fetch `chapterProgress` timestamp and the sort so found this solution to be better.

Reply:
Or can I add the items always at 0th index, so that every recent item will be listed 1st. 
For ex.  `recommendedStoryBuilder.addRecentlyPlayedStory(0,promotedStory)`

Reply:
Ack.

Reply:
The problem, though, is that this is actually duplicating information which means we now have to keep two different things in sync. If we can, we should try to derive the timestamp (which seemed straightforward per this discussion thread: https://github.com/oppia/oppia-android/pull/2272/files/93e64770687cfa8249736b470c17713a489f2966#r554305329).

Reply:
Still unresolved (the value is still in the proto).

Reply:
Proto field is now removed--this is considered resolved from my perspective.